version/1/4/heb/code_of_conduct.md

```diff
@@ -0,0 +1,65 @@
+# קוד התנהגות לתורם
+
+## ההתחייבות שלנו
+
+לטובת עידוד של סביבה פתוחה ומקבלת, אנחנו בתור התורמים והמתחזקים מתחייבים להפוך 
+את ההשתתפות בפרויקט שלנו ואת הקהילה שלנו לחוויה נטולת הטרדות לכל אדם באשר הוא
+ללא הבדלי גיל, משקל, מגבלות פיזיות, מוצא אתני, הזדהות מגדרית, רמת ניסיון, לאום, 
```

Comment:
משקל is "weight"; it is supposed to be the translation of "body size". While it is not far in semantics, alternatives would be:

- `גודל גוף` (body size)
- `מבנה גוף` (body structure)

both would be better translations; the first is also the exact literal translation, the latter a more common reference in Hebrew.